pkg/windows/dockerfile

```diff
@@ -13,9 +13,9 @@ ENV CHOCO_URL=https://chocolatey.org/install.ps1
 #All the variables used for salt
 ENV SALT_SRC_PATH='C:/temp/salt/'
 ENV SALT_GIT_URL=https://github.com/saltstack/salt
-ENV SALT_CHECKOUT=v2018.3.0
+ENV SALT_CHECKOUT=v2018.11
```

Comment:
Has this been tested to work?  I know in previous versions of salt (after 2018.3.0 but before 2018.11) they had changed the internal workings of their modules enough for it to not install through docker.

If you tested this, and it works good, then we don't need to worry